core/cloudflow-blueprint/src/main/scala/cloudflow/blueprint/deployment/ApplicationDescriptorJsonFormat.scala

```diff
@@ -35,7 +35,7 @@ trait ConfigJsonFormat extends DefaultJsonProtocol {
 trait ApplicationDescriptorJsonFormat extends StreamletDescriptorFormat with ConfigJsonFormat {
   implicit val streamletFormat = jsonFormat(StreamletInstance.apply, "name", "descriptor")
 
-  implicit val topicFormat    = jsonFormat(Topic.apply, "id", "config")
+  implicit val topicFormat    = jsonFormat(Topic.apply, "id", "cluster", "config")
```

Comment:
Since it changes the format, you have to increment the `ApplicationDescriptor.Version` and `domain.SupportedApplicationDescriptorVersion` in the cli, so that users using previous versions of kubectl cloudflow are told to update.

Reply:
:+1: